### ISSUE TITLE
[Crash] Freeing any PhysicsBody3D with a CollisionPolygon3D will crash the game when "Visible Collision Shapes" is enabled

### ISSUE THREAD

**Issue by tonylindorock** [godot-jolt/godot-jolt#818]
**Description**
Freeing any PhysicsBody3D with a CollisionPolygon3D (6+ points) will crash the running scene when "Visible Collision Shapes" is enabled.

**Notes**
- If the body is not freed, it can also crash when you close the window.
- This doesn't occur with Godot Physics.

**Repro steps**
1. Create an empty 3D scene.
2. Add a StaticBody3D with a CollisionPolygon3D.
3. Add 6 points to the CollisionPolygon3D.
4. Enable Debug > "Visible Collision Shapes".
5. Run the scene.
6. Close the window or add a script to free the body in _ready.
7. Observe the crash.

**Repro project**
[jolt_polygon_crash.zip](https://github.com/godot-jolt/godot-jolt/files/14736303/jolt_polygon_crash.zip)
I didn't include Jolt in this project because it exceeded the size limit.

**Specs**
- MacBook Pro M1 Pro
- MacOS 14.2.1

**Log**
```
Godot Engine v4.2.1.stable.official.b09f793f5 - https://godotengine.org
Vulkan API 1.2.231 - Forward+ - Using Vulkan Device #0: Apple - Apple M1 Pro
 

================================================================
handle_crash: Program crashed with signal 11
Engine version: Godot Engine v4.2.1.stable.official (b09f793f564a6c95dc76acc654b390e68441bd01)
Dumping the backtrace. Please include this when reporting the bug to the project developer.
[1] 1   libsystem_platform.dylib            0x0000000189a51a24 _sigtramp + 56
[2] 2   godot-jolt_macos_editor             0x000000010fcf8b24 godot_jolt_main + 481236
[3] 3   godot-jolt_macos_editor             0x000000010fcf8b24 godot_jolt_main + 481236
[4] 4   godot-jolt_macos_editor             0x000000010fcccddc godot_jolt_main + 301708
[5] 5   godot-jolt_macos_editor             0x000000010fc9d02c godot_jolt_main + 105692
[6] RenderSceneBuffersConfiguration::set_texture_mipmap_bias(float)
[7] ResourceLoaderText::_parse_sub_resource_dummys(void*, VariantParser::Stream*, Ref<Resource>&, int&, String&)
[8] RendererCompositorRD::_create_current()
[9] JSON::get_data() const
[10] JSON::get_data() const
[11] JSON::get_data() const
[12] Object::can_translate_messages() const
[13] Node::get_index(bool) const
[14] Node::get_index(bool) const
[15] RendererCompositorRD::_create_current()
[16] void rx::mtl::GetMatrixUniformMetal<unsigned int>(unsigned int, unsigned int*, unsigned int const*, bool)
[17] RendererCompositorRD::_create_current()
[18] 18  dyld                                0x00000001896a10e0 start + 2360
-- END OF BACKTRACE --
================================================================
```

**Comment by mihe** [comment]
Thank you for the detailed report, and apologies for the late triage.

I'm not able to reproduce this crash on my Windows machine, regardless if I use the latest `master` or the pre-built 0.12.0-stable, and unfortunately my macOS machine is not up and running right now.

I'm assuming you're running with the pre-built version of Godot Jolt (0.12.0-stable) that's available on the Asset Library? If so, could you try downloading [the debug symbols](https://github.com/godot-jolt/godot-jolt/releases/download/v0.12.0-stable/godot-jolt_v0.12.0-stable_symbols.zip) that are available for that version here on GitHub and just merge that `addons` folder with your existing one. That should hopefully give you a much better backtrace for the Godot Jolt parts of it, and give us a better idea of what the crash is about.

**Comment by tonylindorock** [comment]
Sorry! I almost forgot. It turns out to be way simpler.

It appears that I never updated to 0.12.0 because I too wasn't getting the crash with 0.12.0. So just to test that I wasn't insane, I tried 0.11.0 and got the crash.

Looking back at my repository, I seemed to download 0.11.0 on Jan 7, the same day but maybe just before 0.12.0 was released. So I never realized that I was using an older version. -_- So sorry for this.
